storage/local/state.ts

```diff
@@ -24,6 +25,10 @@ export type State = {
   }
 }
 
+export type AccountToken = Token & {
+  balance: HexString
+}
+
```

Comment:
nit: It is called `AccountToken` and could be placed to account section

Reply:
Yes! my bad, I ignore that section...